tests/src/Kernel/fixtures/markup_rendering_patterns/banner.yml

```diff
@@ -17,9 +17,42 @@ default_page_banner_centered_with_link_full_widht:
   assertions:
     count:
       'div[data-drupal-selector=banner]': 1
-      'div.bg-light.text-dark.bcl-banner': 1
+      'div.bg-lighter.text-dark.bcl-banner': 1
       'div.full-width': 1
       'div.text-center': 1
+      'div.bcl-banner__content': 1
+      'a': 1
+      'div.hero': 0
+      'div.bcl-banner__image': 0
+    contains:
+      'bcl-banner__content': "EU Budget for the future"
+      'p': "Innovation, economy, environment and geopolitics"
+    equals:
+      'a': "Subscribe"
+default_page_banner_centered_with_link_full_widht_with_content_classes:
+  render:
+    '#type': pattern
+    '#id': banner
+    '#variant': 'default'
+    '#fields':
+      settings:
+        'hero': false
+        'shade': false
+        'full_width': true
+        'centered': true
+      'title': "EU Budget for the future"
+      'description': "Innovation, economy, environment and geopolitics"
+      'link':
+        'label': "Subscribe"
+        'path': "https://example.com"
+      'content_classes': "content-class my-2"
+  assertions:
+    count:
+      'div[data-drupal-selector=banner]': 1
+      'div.bg-lighter.text-dark.bcl-banner': 1
```

Comment:
Watching at the example in the netlify (https://oelibrary.netlify.app/default/?path=/story/paragraphs-banners--default&args=title:EU+Budget+for+the+future;label:Subscribe;centered:true;hero:false;full_width:true) here you could add also the text-center and full-width.